fish_prompt.fish

```diff
@@ -82,11 +82,10 @@ function fish_prompt
     # Check if there is an upstream configured
     command git rev-parse --abbrev-ref '@{upstream}' >/dev/null ^&1; and set -l has_upstream
     if set -q has_upstream
-      set -l git_status (command git rev-list --left-right --count 'HEAD...@{upstream}' | sed "s/[[:blank:]]/ /" ^/dev/null)
+      set -l git_status (string split ' ' (string replace -ar '\s+' ' ' (command git rev-list --left-right --count 'HEAD...@{upstream}')))
```

Comment:
Alternatively:
```shell
command git rev-list --left-right --count 'HEAD...@{upstream}' | read -la git_status
```
You will get the first number in `$git_status[1]` and the second in `$git_status[2]`. `read -a` already ignores the extra white-spaces and puts each white-space separated token into a separate array element.